cmd/promtool/unittest.go

```diff
@@ -498,7 +504,7 @@ type labelAndAnnotation struct {
 }
 
 func (la *labelAndAnnotation) String() string {
-	return "Labels:" + la.Labels.String() + " Annotations:" + la.Annotations.String()
+	return "Labels:" + la.Labels.String() + "\n  Annotations:" + la.Annotations.String()
```

Comment:
```suggestion
	return "Labels:" + la.Labels.String() + "\nAnnotations:" + la.Annotations.String()
```

Reply:
are those 2 spaces needed?

Reply:
Oops, they were part of the parent formatting, rearranged it so `labelAndAnnotation` doesn't have knowledge of `labelsAndAnnotations` formatting.